big-practice/src/style/layout/header.css

```diff
@@ -1,21 +1,43 @@
 /* START HEADER */
 /* STYLE APPLIED TO THE MOBILE */
-.logo-bar {
+.header-logo {
   position: fixed;
-  top: 0;
+  z-index: 10;
   width: 100%;
   background: var(--secondary-color-50);
   box-shadow: 0px 1px 0px var(--secondary-color-100);
+  display: flex;
+  justify-content: space-between;
 }
 
-.logo-box {
+.logo-wrap {
   padding-left: 25px;
 }
 
+.avt-box {
+  display: none;
+}
+
 /* STYLE APPLIED TO THE DESKTOP */
 @media (min-width: 1024px) {
-  .logo-box {
+  .logo-wrap {
     padding-left: 45px;
   }
+
+  .avt-box {
+    display: flex;
+    width: 149px;
+    padding: 16px 47px 16px 0;
+    text-decoration: none;
```

Comment:
Do we need this style?

Reply:
Needed because I use the <a> tag to wrap I need this attribute to remove the underline below the word